src/helpers/dom/element.js

```diff
@@ -36,6 +36,34 @@ export function getParent(element, level = 0) {
   return parent;
 }
 
+/**
+ * Gets `frameElement` of the specified frame. Returns null if it is a top frame or if script has no access to read property.
+ *
+ * @param {Window} frame Frame from which should be get frameElement in safe way.
+ * @returns {HTMLIFrameElement|null}
+ */
+export function getFrameElement(frame) {
+  return Object.getPrototypeOf(frame.parent) && frame.frameElement;
+}
+
+/**
+ * Gets parent frame of the specified frame. Returns null if it is a top frame or if script has no access to read property.
+ *
+ * @param {Window} frame Frame from which should be get frameElement in safe way.
+ * @returns {Window|null}
+ */
+export function getParentWindow(frame) {
+  return getFrameElement(frame) && frame.parent;
+}
+
+/**
+ * Checks if script has access to read from parent frame of specified frame.
+ * @param {Window} frame Frame from which should be get frameElement in safe way.
```

Comment:
```suggestion

 * @param {Window} frame Frame from which should be get frameElement in safe way.
```
Added missing line after the description.